src/topology/bounded_continuous_function.lean

```diff
@@ -531,7 +531,7 @@ instance : add_comm_group (α →ᵇ β) :=
   ..bounded_continuous_function.has_sub,
   ..bounded_continuous_function.has_zero }
 
-@[simp] lemma coe_sub : ⇑(f - g) = λ x, f x - g x := rfl
+@[simp] lemma coe_sub : ⇑(f - g) = (⇑f : α → β) - (⇑g : α → β) := rfl
```

Comment:
Does this really need the annotations? I'd expect you can avoid them somehow.

Reply:
```suggestion
@[simp] lemma coe_sub : ⇑(f - g) = f - g := rfl
```

Reply:
I too was surprised but Lean says:
```
unexpected argument at application
  ⇑g
given argument
  g
expected argument
  f - g
```
without them.